components/markdown_image/markdown_image.tsx

```diff
@@ -202,7 +228,12 @@ export default class MarkdownImage extends React.PureComponent {
                                 alt={alt || safeSrc}
                                 postId={postId}
                                 imageKey={safeSrc}
-                                onToggle={onImageHeightChanged}
+                                isExpanded={false}
+                                actions={{
+                                    toggleInlineImageVisibility(): void {
+                                        throw new Error('Function not implemented.');
+                                    },
+                                }}
```

Comment:
from where did these came from, these should not be passed here, since they are dispatched inside of markdownimageexapand itself